From ccd54d70cc4da770302e80d3d171333bd95e389d Mon Sep 17 00:00:00 2001
From: Min Lin <linmin@eswincomputing.com>
Date: Mon, 23 Jun 2025 10:06:00 +0800
Subject: [PATCH 574/652] fix:usb: dwc3: eswin: Fix section mismatch

Changelogs:
dwc3_eswin_deassert() may be called at runtime from dwc3_eswin_probe().

Signed-off-by: Samuel Holland <samuel.holland@sifive.com>
---
 drivers/usb/dwc3/dwc3-eswin.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/dwc3/dwc3-eswin.c b/drivers/usb/dwc3/dwc3-eswin.c
index 0f4231b585a9..edd51751d176 100755
--- a/drivers/usb/dwc3/dwc3-eswin.c
+++ b/drivers/usb/dwc3/dwc3-eswin.c
@@ -255,7 +255,7 @@ static int dwc3_eswin_get_extcon_dev(struct dwc3_eswin *eswin)
 	return 0;
 }
 
-static int __init dwc3_eswin_deassert(struct dwc3_eswin *eswin)
+static int dwc3_eswin_deassert(struct dwc3_eswin *eswin)
 {
 	int rc;
 
-- 
2.54.0

